dependencies.yaml

```diff
@@ -68,6 +71,46 @@ files:
       - depends_on_ogb
       - py_version
       - test_python_common
+  test_cugraph_dgl:
+    output: none
+    includes:
+      - cuda_version
+      - depends_on_cugraph
+      - depends_on_cudf
+      - depends_on_dgl
+      - depends_on_pytorch
+      - depends_on_ogb
+      - py_version
+      - test_python_common
+      - depends_on_pylibwholegraph
+      - depends_on_cugraph_dgl
+  test_cugraph_pyg:
+    output: none
+    includes:
+      - cuda_version
+      - depends_on_cugraph
+      - depends_on_cudf
+      - depends_on_dgl
+      - depends_on_pytorch
+      - depends_on_ogb
+      - py_version
+      - test_python_common
+      - depends_on_pylibwholegraph
+      - depends_on_cugraph_pyg
+  test_pylibwholegraph:
+    output: none
+    includes:
+      - cuda_version
+      - depends_on_cugraph
+      - depends_on_cudf
+      - depends_on_dgl
+      - depends_on_pytorch
+      - depends_on_ogb
```

Comment:
I don't think this is what you want, to replace `conda install ogb`.

`depends_on_ogb`... doesn't seem to contain `ogb`:

https://github.com/rapidsai/cugraph-gnn/blob/2f8afdea8fcb46eae7b1a1d1ef74a37df5cb831d/dependencies.yaml#L438-L464

That looks like a mistake, not sure where it happened. @tingyu66 could you please take a look? Maybe that was the result of a bad merge conflict resolution or something. `depends_on_ogb` seems to just contain `torch` (which we already have `depends_on_pytorch` for)

Reply:
This is a temporary constraint of PyTorch that only applies to test environments where ogb is being used.

Reply:
Bradley and I discussed this here: https://github.com/rapidsai/cugraph-gnn/pull/104

Reply:
And on Slack

Reply:
A better name for this might be `ogb_pytorch_constraint`.

Reply:
Got it, thanks! The comment directly about this, about this being there for the benefit of `ogb`, now makes sense to me.

It still would be better to either name it `ogb_pytorch_constraint`, I think, but that doesn't have to hold up this PR.

Reply:
Ok, also devcontainer builds will fail until #148 gets merged.  I expect that to pass CI now that the cudf fix is merged.

Reply:
Now that that's merged, I've merged in latest `branch-25.04` here to re-run CI.